sorc/emcsfc_snow2mdl.fd/grib_utils.f

```diff
@@ -142,40 +120,28 @@ SUBROUTINE SKGB2(LUGB,ISEEK,MSEEK,LSKIP,LGRIB,I1)
  RETURN
  END subroutine skgb2
 
- subroutine gdt_to_gds(igdtnum, igdstmpl, igdtlen, kgds, ni, nj, res)
-!$$$  subprogram documentation block
-!
-! subprogram:    gdt_to_gds
-!   prgmmr: gayno          org: w/np2     date: 2014-sep-26
-!
-! abstract:  convert from the grib2 grid description template array
-!            used by the ncep grib2 library, to the grib1 grid
-!            description section array used by ncep ipolates library.
-!  
-! program history log:
-! 2014-sep-26  gayno    - initial version
-!
-! usage: call gds_to_gds(igdtnum,igdstmpl,igdtlen,kgds,ni,nj,res)
-!
-!   input argument list:  
-!     igdtnum  - grib2 grid desc template number
-!     igdstmpl - grib2 grid desc template array
-!     igdtlen  - grib2 grid desc template array size
+!> Convert from the grib2 grid description template array
+!! used by the ncep grib2 library, to the grib1 grid
+!! description section array used by ncep ipolates library.
+!!  
+!! program history log:
+!! - 2014-Sep-26  gayno    - initial version
+!!
+!! @param[in] igdtnum grib2 grid desc template number
+!! @param[in] igdstmpl grib2 grid desc template array
+!! @param[in] igdtlen grib2 grid desc template array size
 !
```

Comment:
I am getting some doxygen warnings on this routine.  Is it due to the single "!"?